src/libcompiler_builtins/Cargo.toml

```diff
@@ -13,3 +13,4 @@ core = { path = "../libcore" }
 
 [build-dependencies]
 gcc = "0.3.27"
+rustc-cfg = { git = "https://github.com/japaric/rustc-cfg", branch = "llvm-target" }
```

Comment:
The rationale is to make this crate more robust at handling custom targets whose triple may be `foo` or any other arbitrary name. *My* main motivation was to handle Cortex-M targets which are currently customs target and may be named whatever by downstream users. But if those Cortex-M targets land in tree then I'm OK with not landing this PR.